backend/kernelCI_app/helpers/tests.py

```diff
@@ -0,0 +1,27 @@
+import json
+from typing import Union, TypedDict, Optional
+from kernelCI_app.helpers.filters import UNKNOWN_STRING
```

Comment:
`filters` is probably not the best place for `UNKNOWN_STRING` maybe `helper.constants` 

but I know it is not being introduced by this PR

Reply:
I agree with you. I can change this as well if you think it would be better

Reply:
lets not worry for this for today, we need this quick